packages/plugins/test/mocha/foundry/utils.spec.ts

```diff
@@ -733,7 +733,8 @@ describe('Utils', async () => {
       const broadcast = await runForgeScript(
         'contracts/test/script/Libraries.s.sol',
         foundryToml.broadcastFolder,
-        getAnvilRpcUrl(chainId)
+        getAnvilRpcUrl(chainId),
+        'MyContractWithLibraries_Script'
       )
```

Comment:
After switching back to the official foundry release, I noticed this test was failing with the following error:
`Could not find target contract: contracts/test/script/Libraries.s.sol`

Since the changes in this PR are relatively minor, and because none of our other tests broke, this is likely a regression in Foundry. So I just fixed this by updating the `runForgeScript` function to accept a target contract function.